package/libpiksi/libpiksi/src/endpoint.c

```diff
@@ -203,7 +189,8 @@ static int pk_endpoint_receive_nn_msg(pk_endpoint_t *pk_ept, void *buffer_loc, s
   while (1) {
     length = nn_recv(pk_ept->nn_sock, buffer_loc, *length_loc, nonblocking ? NN_DONTWAIT : 0);
     if (length >= 0) {
-      if (length == 0) piksi_log(LOG_WARNING, "Empty message received");
+      if (length == 0)
+        piksi_log(LOG_WARNING, "Empty message received");
```

Comment:
This is not respecting our standard of "if statement with one statment should all be on one line".

Reply:
Yea I missed that, I'll have to set `AllowShortIfStatementsOnASingleLine` to true